tests/unit/Metadata/Parser/AttributeParserTestCase.php

```diff
@@ -53,6 +54,7 @@
 use PHPUnit\TestFixture\Metadata\Attribute\UsesTest;
 use PHPUnit\TestFixture\Metadata\Attribute\WithoutErrorHandlerTest;
 
+#[CoversClass(AttributeParser::class)]
```

Comment:
This has no effect as `AttributeParserTestCase` is an abstract class and only metadata on concrete test classes is used.

Reply:
thanks for the feedback. is this something phpunit should error about?

Reply:
Good questions. Yes, it probably should. However, figuring out that there is something to warn about would require additional work at runtime for each test. I am not convinced that this is worth it.

Reply:
agree it might be not worth it. fwiw we already do some heavy lifting using reflection in this case:

https://github.com/sebastianbergmann/phpunit/blob/f6c0e6d7c80e07b37886b05c6e738b5fd760368c/src/Metadata/Api/CodeCoverage.php#L315-L323

Reply:
That is code for classes targeted by `#[CoversClass]` and `#[UsesClass]`, not for test classes that use these attributes.